recipes/libpq/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "9.5.3" %}
+
+package:
+    name: libpq
+    version: {{ version }}
+
+build:
+  number: 0
+  skip: True   # [win]
+  # features:
+  #   - vc9               # [win and py27]
+  #   - vc10              # [win and py34]
+  #   - vc14              # [win and py35]
+  always_include_files:
```

Comment:
Not new, but not commonly used.  This is what you do when you want to create a sort of sub-package, as I did here with libpq.  The files originally come from postgresql, but then I force them to be included in the new package (they would not be, otherwise, since they exist before the build starts.)